docs/development.rst

```diff
@@ -55,7 +55,7 @@ to **activate** the virtual environment:
 
    $ . ./bin/activate
 
-The command-prompt will change to indicate that you are now in a
```

Comment:
If  the virtual-environment activation changes `$PS1` to something other than the default then there will be a visible change, but I can imagine cases where it will stay the same. Maybe *should* would be a better word (if we are going to get all RFC2119 on this 😉).